public/app/features/dashboard/components/DeleteDashboard/DeleteDashboardModal.tsx

```diff
@@ -34,7 +35,15 @@ const DeleteDashboardModalUnconnected = ({ hideModal, cleanUpDashboardAndVariabl
       source: 'dashboard_settings',
       restore_enabled: config.featureToggles.dashboardRestoreUI,
     });
-    await deleteDashboard(dashboard.uid, true);
+    // await deleteDashboard(dashboard.uid, true);
+    await deleteItems({
+      selectedItems: {
+        dashboard: {
+          [dashboard.uid]: true,
+        },
+        folder: {},
+      },
+    });
```

Comment:
This only works outside of scenes. In scenes the `useDeleteItemsMutation()` hook cannot be used.